Tribler/Core/Utilities/utilities.py

```diff
@@ -14,7 +14,7 @@
 from libtorrent import bdecode, bencode
 
 import six
-from six.moves.urllib.parse import parse_qsl, urlsplit
+from six.moves.urllib.parse import parse_qsl, quote_plus, urlsplit
```

Comment:
Why are you not using `quote_plus_unicode` anymore? If you simply need `quote_plus`, you did not have to move the utilities to the core?

Reply:
There are two instances where `quote_plus_unicode` is used. One in Core in this file `utilities.py` and the other in `startdownloaddialog.py`. If looking for a minimal change, it would be to move the `compose_magnetlink()` of core `utilities.py`  method to the GUI. But I think that function fits to the Core.